File: applied/05_top_of_the_world/answer/2.c
// n個の数字を入力し、それらを合計して出力しよう
// 0が入力されたら入力を終了します

#include <stdio.h>
int main(void){
    int input, acc;
    scanf("%d", &input);

    for (acc = 0; input != 0; scanf("%d", &input)) {
        acc += input;
    }
    
    printf("%d\n", acc);
}

File: applied/05_top_of_the_world/answer/2.java
// n個の数字を入力し、それらを合計して出力しよう
// 0が入力されたら入力を終了します

import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        int input, acc;

        for (acc = 0, input = scan.nextInt(); input != 0; acc += input, input = scan.nextInt()) ;
    
        System.out.printf("%d\n", acc);
    }
}
